backend/schedulers/monthly_dump_enqueue_scheduler.py

```
@@ -0,0 +1,56 @@
+            return
+
+        self.scheduler.shutdown(wait=True)
+        self.is_running = False
+        logger.info("Monthly dump enqueue scheduler stopped", {"job_id": "enqueue_monthly_dump_jobs"})
+
+    async def _enqueue_job(self) -> None:
+        try:
+            # We don't need to await because the service uses sync supabase calls
+            self.enqueue_service.enqueue_eligible_users()
+        except Exception as exc:  # noqa: BLE001
+            logger.error(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)